packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -9,101 +9,120 @@ ouia: true
 import pfLogo from './pfLogo.svg';
 import pfLogoSmall from './pf-logo-small.svg';
 
-## Examples
 
-### Basic
+## Examples 
+
+### Basic cards
+
+Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`. You may omit these components as needed, but it is recommended to at least include a `<CardBody>` to provide details about the card item.
 
 ```ts file='./CardBasic.tsx'
 ```
 
-### With modifiers
+### Modifiers
+
+You can further modify the styling of the card's content by using the properties in the following example. In the example, you can enable each modifier by selecting its respective checkbox.
+
+Most modifiers can be used in combination with each other, with the exception of `isCompact` and `isLarge`, since they are contradictory.
 
 ```ts file='./CardWithModifiers.tsx'
 ```
 
-### With image and actions
+| Modifier | Description |
+| --- | --- |
+| isCompact | Modifies the card to include compact styling. Should not be used with isLarge. |
+| isFlat | Modifies the card to include flat styling. |
+| isRounded | Modifies the card to include rounded border styling. |
+| isLarge | Modifies the card to be large. Should not be used with isCompact. |
+| isFullHeight | Modifies the card so that it fills the total available height of its container. |
+| isPlain | Modifies the card to include plain styling, which removes the border and background. | 
+
+
+### Header images and actions
+
+You can include header images using `<Brand>` within `<CardHeaderMain>` and you can include header actions within `<CardActions>`.
 
 ```ts file='./CardWithImageAndActions.tsx'
 ```
 
-### Header in card head
+### Title inline with images and actions
+
+Moving `<CardTitle>` within the `CardHeader` component will style it in line with any images or actions.
 
 ```ts file='./CardHeaderInCardHead.tsx'
 ```
 
 ### Only actions in card head (no header/footer)
 
-```ts file='./CardOnlyActionsInCardHead.tsx'
-```
-
-### Only image in card head
-
-```ts file='./CardOnlyImageInCardHead.tsx'
-```
+You may place `<CardActions>` in `<CardHeader>` even if you're only using `<CardBody>`.
 
-### With no footer
-
-```ts file='./CardWithNoFooter.tsx'
+```ts file='./CardOnlyActionsInCardHead.tsx'
 ```
 
-### With no header
-
-```ts file='./CardWithNoHeader.tsx'
-```
+### With HTML heading element
 
-### With only a body section
+You may use the `component` property to place the card's title within an HTML heading element.
 
-```ts file='./CardWithOnlyBodySection.tsx'
+```ts file='./CardWithHeadingElement.tsx'
 ```
 
 ### With multiple body sections
 
+You may use multiple body sections to visually separate different sections of content.
+
 ```ts file='./CardWithMultipleBodySections.tsx'
 ```
 
-### With only a body section that fills
+### With a body section that fills
+
+Use the `isFilled` property to let the body content fill the height of the card.
```

Comment:
Since `isFilled` is "true" by default, and to explain a little more what is going on in the example, maybe something like:

```suggestion
The `isFilled` property allows body content to fill the height of the card, after taking into account the height that is taken up by any other card content.

The following example has 3 `<CardBody>` sub-components, with two having `isFilled` set to "false" to show how the third card body fills the remaining height.
```

@mcoker do you have a better way to convey what is going on in this example in terms of how the `isFilled` prop and `pf-m-no-fill` classes are interacting?

Reply:
> Since `isFilled` is "true"

Good call out. It's actually the case that card body sections grow to fill the available height of the card by default. This is only visible if the card is taller than the content inside of it. In that case, the body(s) will grow to fill any available space, and they all grow together. `isFilled` only really exists so you can set it to `false`, which disabled that feature on a body element. Probably the most common use case is to have multiple body elements, and set all but one of them to `isFilled={false}` to have a primary body fill the available space of the card.

Here's a demo that shows how that works - https://codepen.io/mcoker/pen/qBoQyVN

I might say something like "Card body sections are setup to fill the available height/space of the card by default via `isFilled={true}`. To disable that for one or more body sections, set `isFilled={false}`.

And maybe title the example "With a primary body section using `isFilled`"?

Reply:
I opted to call this "With a primary body section that fills" since we haven't used property names in example headings yet (that I've noticed), but if that's not descriptive enough then we can swap "fills" over to `isFilled`

Reply:
Not sure if PatternFly will have an issue with it, but outside of PF I believe I've noticed that using codeblocks in headings can be iffy when it comes to creating links to/rendering them. So if we did want to include `isFilled` in the example title, maybe instead just "With a primary body section using isFilled" without the backticks to be safe?

I'm good with either that or the current example title, though. I think naming this one succinctly may just be a little more difficult since there's a bit more going on than a short title may be able to convey.